snapd-glib/snapd-notice.c

```diff
@@ -170,6 +189,25 @@ snapd_notice_get_first_occurred (SnapdNotice *self)
     return g_date_time_ref (self->first_occurred);
 }
 
+/**
+ * snapd_notice_get_first_occurred2: (rename to snapd_notice_get_first_occurred)
+ * @notice: a #SnapdNotice.
+ *
+ * Get the time this notification first occurred.
+ *
+ * Returns: (transfer none): a #GDateTime.
+ *
+ * Since: 1.66
+ */
+GDateTime *
+snapd_notice_get_first_occurred2 (SnapdNotice *self)
+{
+    g_return_val_if_fail (SNAPD_IS_NOTICE (self), NULL);
+    if (self->first_occurred == NULL)
+        return NULL;
```

Comment:
This doesn't do anything - it would have returned NULL in the next line anyway!

Reply:
Ops!!!! Yes, it was the check to avoid errors when calling XXX_ref. The problems of copy&paste 😅